### PR TITLE
Global exception handler and data validation

### PR DESCRIPTION
**Pull request by OlegMR0**

### CODE REVIEW

**Comment by OlegMR0** [comment]
Please help me to fix.
It should work, but output in postman is the same
image

File: src/main/java/com/example/bookstore/exception/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @Override
```
Comment (okuzan) [inline src/main/java/com/example/bookstore/exception/GlobalExceptionHandler.java:24]: you can also add
```
    @ExceptionHandler(EntityNotFoundException.class)
    protected ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundException ex) {
        Map<String, Object> body = new LinkedHashMap<>();
        body.put("timestamp", LocalDateTime.now().format(FORMATTER));
        body.put("status", HttpStatus.NOT_FOUND);
        body.put("error", "Entity Not Found");
        body.put("message", ex.getMessage());
        return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
    }
```

**Review by okuzan — CHANGES_REQUESTED** [review]
Need further details

**Review by Ivan95kos — APPROVED** [review]
Great job

**Pull request merged**